### ISSUE TITLE
orphaned spinner for `launch inject`

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2509]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.31.1](https://github.com/open-wa/wa-automate-nodejs/compare/4.31.0...4.31.1) (2022-02-17)

- Release 4.31.1 [`2befe72`](https://github.com/open-wa/wa-automate-nodejs/commit/2befe727cb9f14ed6f5a5333fc1373ff859677fd)
- 🏷️  fix `getUnreadMessages` typing #2417 [`281c3cf`](https://github.com/open-wa/wa-automate-nodejs/commit/281c3cf8642910543cc59d5c2c0806756cad780c)
- 🔊 prevent orphaned launch inject spinner #2509 [`47dcc31`](https://github.com/open-wa/wa-automate-nodejs/commit/47dcc31f06f38da19ca6ca8f1c9ff02390fd1a27)
- 🩹  fix launch patch [`16cd42e`](https://github.com/open-wa/wa-automate-nodejs/commit/16cd42e5d70bfd303c30e33832ec2355b2359aa9)
- updated types-only package [`a66bd2d`](https://github.com/open-wa/wa-automate-nodejs/commit/a66bd2dbd081605d9fe974ebe1565d6a732b5863)
